docs/examples/tracking-nrel.py

```diff
@@ -0,0 +1,55 @@
+"""
+Flag Sunny Days for a Tracking System
+=====================================
+
+Flag sunny days for a single-axis tracking PV system.
+"""
+
+# %%
+# Identifying and masking sunny days for a single-axis tracking system is
+# important when performing future analyses that require filtered clearsky
+# data. For this example we will use data from the single-axis tracking
```

Comment:
Wording nitpick -- I think it's probably a good idea to avoid using "clear-sky" and "sunny day" as synonyms.  Clear-sky (at least in the pvlib world) has some association with smoothness, but I think these functions would not take issue with a jagged input curve so long as the jaggedness is riding on top of the quartic/quadratic profile they're looking for. 

The tracking function has the additional requirement that the input *isn't* a good match to a fixed-tilt profile, so to my mind it's more of a check that the system is tracking properly, and the identification of sunny days was more of an implementation detail than a specific goal.  Not sure how relevant that is here though.

Reply:
I changed the wording to 'sunny day' only in https://github.com/pvlib/pvanalytics/pull/148/commits/76c4de1b0b66a64a618800c58fe174ace294d98e